````
icecream-cpp is header-only and can be used from CMake via:

  find_path(ICECREAM_CPP_INCLUDE_DIRS "icecream.hpp")
  target_include_directories(main PRIVATE ${ICECREAM_CPP_INCLUDE_DIRS})
````
@toge, Thanks for your PR, have you tested this usage locally? Why do I get an error when using it:
````
1> [CMake] CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
1> [CMake] Please set them or make sure they are set and tested correctly in the CMake files:
1> [CMake] C:/Users/TEMP/source/repos/CMakeProject1/CMakeProject1/ICECREAM_CPP_INCLUDE_DIRS
1> [CMake]    used as include directory in directory C:/Users/TEMP/source/repos/CMakeProject1/CMakeProject1
````

--- a/ports/icecream-cpp/portfile.cmake
+++ b/ports/icecream-cpp/portfile.cmake
@@ -22,5 +22,4 @@ else()
 endif()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib")
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.txt")

--- a/versions/i-/icecream-cpp.json
+++ b/versions/i-/icecream-cpp.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "d2e66987c8dac2451dfaf1ab43023b8537ee853e",
+      "git-tree": "f45a86c34d4fa023eb0be088e187676628843860",
       "version": "1.0.0",
       "port-version": 0
     }

